### ISSUE TITLE
export clause doesn't apply its argument to current project

### ISSUE THREAD

**Issue by xahon** [Meoo/premake-export#2]
For example
```Lua
project("Lib")
   export({"Stuff"})
      defines({"IT_IS_WORKING"})
   export({})
```

```Lua
project("Exe")
   import({
      ["Lib"] = {"Stuff"},
   })
```

`Exe` project works as expected and contains define directive `IT_IS_WORKING` 
but `Lib` doesn't  
I expect both projects to contain `IT_IS_WORKING` directive  
A workaround like that doesn't work too  
```Lua
project("Lib")
   export({"Stuff"})
      defines({"IT_IS_WORKING"})
   export({})
   import({
      ["Lib"] = {"Stuff"},
   })
```

**Comment by xahon** [comment]
@Meoo please, pay attention to the issue. I think it is very easy to fix but I don't know premake api and how to develop plugins for it (and, to be clear, lua in general)

**Comment by Meoo** [comment]
It should be `export("Stuff")` instead of `export({"Stuff"})`

If it's not enough to fix it then it's probably an incompatibility with newer versions of premake.
Also, please note that this project is not maintained anymore as I stopped using premake

**Comment by xahon** [comment]
@Meoo no, it doesn't work too. Looks like it is not the reason because export clause works for other (not current) projects
